sky/packages/sky/lib/widgets/scrollable.dart

```diff
@@ -459,7 +460,12 @@ abstract class ScrollableWidgetList extends Scrollable {
         } else {
           drawStart = paddedScrollOffset;
         }
-        itemShowIndex = math.max(0, (drawStart / itemExtent).floor());
+        if (itemCount > 0) {
+          // floor(epsilon) = 0, floor(-epsilon) = -1, so:
+          if (drawStart.abs() < _kEpsilon)
+            drawStart = 0.0;
```

Comment:
The arithmetic for scrolling towards zero doesn't always land exactly on zero, sometimes it lands on, well +/- epsilon. Without this little bit of smoothology, scrolling to zero can overshoot.